.github/workflows/cd.yml

```
@@ -72,6 +76,57 @@ jobs:
           path: dist
           name: dist
 
+  # -----
+  linux-arm64:
```

```suggestion
  linux-aarch64:
```
Let's use same notation as Rust/LLVM, here and anywhere below.

---

I also considered using "aarch64," but for macOS, it is "arm64". Is it OK to have different naming rules for Linux and macOS? Personally, I prefer to use "aarch64" for macOS as well.

---

Changed arm64 to aarch64 on linux side only.

---

Hmm I hadn't checked the macOS build but indeed that's a problem, we need to keep the archive arch names consistent for different OSes. 
But we are already not consistent with it, it seems:
| Go | Rust/LLVM |
| - | - |
| `amd64` | `x86_64` |
| `386` | `i686` | 
| `arm64` | `aarch64` |

I preferred `aarch64` because we are already using the LLVM notation for `x86_64`, so I would prefer the macOS archive & this new Linux archive to use same naming. But that would break existing scripts/build systems that fetch macOS archives.

@almarklein, what does Python build system prefer, and WDYT about changing macOS archive to use `aarch64` instead of current `arm64`.

---

This only really affects the name of the zipfile to download from the release page.

We [construct the name of the zipfile](https://github.com/pygfx/wgpu-py/blob/4478b19d03598036169eaeed46bd05c4ac0dcf29/download-wgpu-native.py#L79-L101) in wgpu-py, and we can easily update this as long as there's some clear logic to follow. 

As far as how Python names it's binary wheels:
* For Windows it's `win_amd64`  or `win32` (not sure what an arm build would be called).
* For Linux it's `x86_64` or `aarch64`.
* For MacOS it's `x86_64`  or `arm64`.

I'd decide whatever makes the most sense from the pov of C/Rust developers.

---

> This only really affects the name of the zipfile to download from the release page.

Yes, also not so much of a problem for users with versioned dependency loading/downloading (which is probably everyone).

> For Windows it's win_amd64 or win32, For Linux it's x86_64 or aarch64, For MacOS it's x86_64 or arm64.

Wild, each OS uses different arch notation. I would prefer it to be consistent though.

So, let's update the macOS build to use `aarch64` too. @dskkato can you please include that too in this PR?

---

I've changed macOS `arm64` to `aarch64`. Please check the results in the CD pipeline on my forked repository.

https://github.com/dskkato/wgpu-native/actions/runs/7041779247